### PR TITLE
vis_utils.plot_positions: add various options

### PR DESCRIPTION
**Pull request by folterj**
vis_utils.plot_positions: added options:

- custom labels
- label size
- whether to show or not
- output filename to save to file

### CODE REVIEW

File: src/multiview_stitcher/vis_utils.py [hunk 1]
```diff
@@ -21,6 +21,10 @@ def plot_positions(
     n_colors=2,
     nscoord=None,
     display_view_indices=True,
+    custom_labels=None,
+    label_size=10,
```
Comment (m-albert) [inline src/multiview_stitcher/vis_utils.py:25]: Agreed this is convenient! Could we rename these to "view_labels" and "view_labels_size"? I feel this makes it slightly more clear.

Comment (folterj) [reply src/multiview_stitcher/vis_utils.py:25]: Absolutely, will change labels_size to view_labels_size and push another commit. Any other suggestions, please let me know, and also feel free to modify anything that makes more sense. :)

Comment (m-albert) [reply src/multiview_stitcher/vis_utils.py]: Perfect, thank you! If you agree, I'd also propose to change `custom_labels` to `view_labels` for consistency.

Comment (folterj) [reply src/multiview_stitcher/vis_utils.py]: Even better, change pushed.

**Review by m-albert — APPROVED** [review]

File: src/multiview_stitcher/vis_utils.py (hunk 1 above)
Comment (m-albert) [reply src/multiview_stitcher/vis_utils.py]: Thank you!

**Pull request merged**